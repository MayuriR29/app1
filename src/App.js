import React, { Component } from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { Layout } from "antd";
import ContentRoutes from "./Component/ContentRoutes";
import NavHeader from "./Component/NavHeader/NavHeader";
import "./App.css";

const { Content, Footer } = Layout;

class App extends Component {
  render() {
    return (
      <div>
        <Layout className="layout">
          <Router>
            <div>
              <NavHeader />
              <Content style={{ padding: "0 50px" }}>
                <ContentRoutes />
              </Content>
              <Footer style={{ textAlign: "center" }}>
                Ant Design Â©2018 Created by Ant UED
              </Footer>
            </div>
          </Router>
        </Layout>
      </div>
    );
  }
}

export default App;
