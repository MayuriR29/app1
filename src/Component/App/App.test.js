import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import ShallowRenderer from 'react-test-renderer/shallow'

it('should render snapshot correctly', () => {
  const renderer=new ShallowRenderer();
  renderer.render(<App/>);
  const output=renderer.getRenderOutput();
  expect(output).toMatchSnapshot();
});
