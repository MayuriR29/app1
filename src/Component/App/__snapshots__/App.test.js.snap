// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shpuld render snapshot correctly 1`] = `
<div>
  <Adapter
    className="layout"
  >
    <BrowserRouter>
      <div>
        <NavHeader />
        <Adapter
          style={
            Object {
              "padding": "0 50px",
            }
          }
        >
          <ContentRoutes />
        </Adapter>
        <Adapter
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          Ant Design Â©2018 Created by Ant UED
        </Adapter>
      </div>
    </BrowserRouter>
  </Adapter>
</div>
`;
